geomesa-jupyter/pom.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>geomesa_2.11</artifactId>
+        <groupId>org.locationtech.geomesa</groupId>
+        <version>1.3.0-m1-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>geomesa-jupyter_2.11</artifactId>
+    <name>GeoMesa Jupyter Extensions</name>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.scala-lang</groupId>
+            <artifactId>scala-library</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.scala-lang.modules</groupId>
+            <artifactId>scala-xml_2.11</artifactId>
```

Comment:
In general `${scala.binary.version}` should be used for dependencies instead of `_2.11`.   A profile should be used if the dependency doesn't exist or isn't needed for one our support versions of Scala.  For this particular dependency, it  isn't needed for `2.10` so it should go in a profile.